### ISSUE TITLE
Trying to load multiple streams on one page fails on initial page load.

### ISSUE THREAD

**Issue by leggettc18** [k-yle/rtsp-relay#38]
Hello,

I'm having an issue while using this while trying to load more than one stream at once on a page. Without fail it will not load the video streams on the first page load, only after a reload. I can also get it to work 90% of the time by adding small timeouts between creating the new JSMpeg Players. Once everything is loaded it works fantastically, but this is a papercut I'd like to fix if possible.

I've added several console.log statements to the JSMpeg players' event handler functions to see what they were doing. I was able to see that all 4 of them on this page establish their connections, start playback, and then don't do anything else. the onPause, onEnded, onStalled, and onVideoDecode, all have console.log statements as their callbacks and none of them ever get called. I have tried passing additional options to ffmpeg through the proxy's `additionalFlags` option, to try and decrease framerate, bitrate, and resolution, and they do take effect when the streams are loaded but do not resolve the issue here.

**Comment by k-yle** [comment]
Hi, this is a bug with JSMpeg itself. The solution is to wait until the first player is ready before creating the next one. But, JSMpeg makes this unnecessarily hard because it doesn't have an `onReady` event. 

I've just released `v1.3.0` which changes how the client-side player is loaded. Instead of loading JSMpeg directly, there is now a function called `loadPlayer` which returns a promise. Once the promise resolves, you can create the next player. Check out the README for more details. 

Usage:
```js
const [canvas1, canvas2] = document.querySelectorAll('canvas');

async function main() {
  await loadPlayer({ url: 'ws://' + location.host + '/api/stream/1', canvas: canvas1 });
  await loadPlayer({ url: 'ws://' + location.host + '/api/stream/2', canvas: canvas2 });
}
main();
```

I've added an integration test for this case which is now passing - Let me know if it works for you.

**Comment by leggettc18** [comment]
Fantastic! That works much better.
